supplementary_style_guide/style_guidelines/specific-doc-types.adoc

```diff
@@ -5,5 +5,104 @@
 [[release-notes]]
 == Release notes
 
+[[release-notes-doc-texts]]
+=== Conventions for phrasing Release Notes doc texts
+
+A _doc text_ is a short description of an engineering Bugzilla or Jira issue that is published in product Errata advisories and Release notes. Engineering supplies draft content, which is a summary of the issue, then technical writers edit or rewrite the draft content in accordance with _The IBM Style Guide_. You can write doc texts in more than one way, depending on the issue type: 
```

Comment:
Release notes -> release notes

Reply:
Engineering supplies -> Engineering typically supplies ?

I'm used to getting a draft, but I wasn't sure if that was always the case.